webapp/templates/base.html.twig

```diff
@@ -45,16 +45,6 @@
         var domjudge_base_url = "{{ app.request.getBaseURL() }}";
     {% endif %}
     $(function () {
-        $('body').on('change', '.custom-file-input', function () {
```

Comment:
Is the file widget now natively supported, or is it now different?

Reply:
The default view is i.m.o. beter and I've moved this functionality of listing all filenames to the place where it matters and solved it there in an even better way.

Reply:
Yeah I just looked at it and it looks good.